library/core/src/cell.rs

```diff
@@ -929,7 +929,7 @@ impl<T: ?Sized> RefCell<T> {
     /// Also, please be aware that this method is only for special circumstances and is usually
     /// not what you want. In case of doubt, use [`borrow_mut`] instead.
     ///
-    /// [`borrow_mut`]: #method.borrow_mut
+    /// [`borrow_mut`]: RefCell::borrow_mut()
```

Comment:
nit: can we try dog-fooding `Self`? It should work here.
```suggestion
    /// [`borrow_mut`]: Self::borrow_mut()
```